### PR TITLE
fixed missed dependencies in far2l.rb

### PR DESCRIPTION
**Pull request by snowytoxa**
added these four:
- uchardet
- minizip
- spdlog
- xerces-c

### CODE REVIEW

**Comment by AlexanderZagaynov** [comment]
works, thank you!

**Comment by unxed** [comment]
minizip is not needed any more, it's usage was removed from far2l's code

File: Formula/far2l.rb [hunk 1]
```diff
@@ -14,6 +14,10 @@ class Far2l < Formula
   depends_on "pkg-config"
   depends_on "wget"
   depends_on "wxmac"
+  depends_on "minizip"
```
Comment (snowytoxa) [inline Formula/far2l.rb]:
```suggestion
```

**Review by gregkonush — APPROVED** [review]

**Comment by andrewpg** [comment]
for osx BigSur it works with combination on this:
https://github.com/elfmz/far2l/issues/866#issuecomment-751979246

**Comment by yurikoles** [comment]
Should be fixed as of 46682a6f8156f90c2473e35cf3d3e68b172675c0

**Pull request closed without merging**